webpages/common/components/inline-icon.html

```diff
@@ -0,0 +1,28 @@
+<template>
+  <svg
+    :class="class"
+    :title="title"
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    fill="currentcolor"
+    style="transform: rotate(360deg)"
+  >
+    <path
+      v-if="icon === 'cloud'"
+      d="M18.42 9.21a7 7 0 0 0-13.36 1.9A4 4 0 0 0 6 19h11a5 5 0 0 0 1.42-9.79zM17 17H6a2 2 0 0 1 0-4 1 1 0 0 0 1-1 5 5 0 0 1 9.73-1.61 1 1 0 0 0 .78.66A3 3 0 0 1 17 17z"
+    />
+    <path
+      v-if="icon === 'evenlope'"
```

Comment:
You spelled envelope wrong here and here

https://github.com/ScratchAddons/ScratchAddons/blob/ba8c3357b28a99bc39c1f20fe58d59b21123f847/addons/scratch-messaging/addon.json#L13